tests/api/testdata.json

```diff
@@ -0,0 +1,121 @@
+{
+  "blocks" : [
```

Comment:
Can you add a "comment" field (shouldn't break anything, should it?) or similar here with a note, where you are taking the test data from (I assume from ``ethereum/tests`` together with a date and HF note?